backend/dynamo-api/src/handlers/publish-to-sns.ts

```diff
@@ -0,0 +1,37 @@
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from 'aws-lambda';
+import { SNSClient, PublishCommand } from "@aws-sdk/client-sns";
+const snsClient = new SNSClient({ region: "eu-west-2" });
+
+export const publishToSNSHandler = async (event: APIGatewayProxyEvent, context: Context): Promise<APIGatewayProxyResult> => {
+    const payload = event?.body ? JSON.parse(event.body as string) : event;
+    console.debug(payload)
+    const emailContent = 'Request to join GOV.UK Sign In private beta form submitted\n\n'+
+                         'Name: '+ payload.name + '\n' +
+                         'Email address: '+ payload.emailAddress + '\n' +
+                         'Service name: '+ payload.serviceName + '\n' +
+                         'Department: ' + payload.department + '\n'
```

Comment:
```suggestion
    const emailContent = "Request to join GOV.UK Sign In private beta form submitted\n\n" +
                         `Name: ${payload.name}\n` +
                         ...
```
Use string interpolation